zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -678,6 +713,135 @@ components:
         endDate:
           type: string
           format: date-time
+    UserTaskSearchQueryRequest:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      description: User task search query request.
+      type: object
+      properties:
+        filter:
+          $ref: "#/components/schemas/UserTaskFilterRequest"
+    UserTaskSearchQueryResponse:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryResponse"
+      description: User task search query response.
+      type: object
+      properties:
+        items:
```

Comment:
This was converted to Strings on this PR 
https://github.com/camunda/camunda/pull/19638